engine/access/rpc/backend/script_executor.go

```diff
@@ -85,6 +90,18 @@ func (s *ScriptExecutor) ExecuteAtBlockHeight(ctx context.Context, script []byte
 		return nil, err
 	}
 
+	// Version control feature could be disabled by cmd argument "--version-control-enabled". In such a case, ignore related functionality.
```

Comment:
let's avoid including details in comments that my get out of date
```suggestion
	// Version control feature could be disabled. In such a case, ignore related functionality.
```